slurm/datadog_checks/slurm/check.py

```diff
@@ -272,9 +277,59 @@ def process_sacct(self, output):
             self.gauge('sacct.job.duration', duration, tags=tags)
             self.gauge('sacct.slurm_job_avgcpu', ave_cpu, tags=tags)
             self.gauge('sacct.job.info', 1, tags=tags)
+            if self.collect_seff_stats:
+                job_state = job_data[12].strip().upper()
+                # Run on Completed Jobs only https://wiki.rcs.huji.ac.il/hurcs/guides/resource-utilization
+                if not has_suffix and job_state == 'COMPLETED':
+                    self.log.debug("Processing seff for job %s", job_id)
+                    self.process_seff(job_id, tags)
 
         self.gauge('sacct.enabled', 1)
 
+    def process_seff(self, job_id, tags):
+        cmd = self.seff_cmd + [str(job_id)]
+        self.log.debug("Running seff command: %s", cmd)
+        out, err, ret = get_subprocess_output(cmd)
+        if ret != 0 or not out:
+            self.log.debug("seff command failed for job %s: %s", job_id, err)
+            return
+
+        cpu_utilized = None
+        cpu_eff = None
+        mem_utilized = None
+        mem_eff = None
+
+        for line in out.splitlines():
+            line = line.strip()
+            if line.startswith('CPU Utilized:'):
+                # CPU Utilized: 00:00:01
+                value = line.split(':', 1)[1].strip()
+                cpu_utilized = parse_duration(value)
+            elif line.startswith('CPU Efficiency:'):
+                # CPU Efficiency: 20.00% of 00:00:05 core-walltime
+                match = re.match(r'CPU Efficiency: ([\d.]+)%', line)
+                if match:
+                    cpu_eff = float(match.group(1))
```

Comment:
```suggestion
            if re.match(r'CPU Efficiency: ([\d.]+)%', line):
                # CPU Efficiency: 20.00% of 00:00:05 core-walltime
                cpu_eff = float(match.group(1))
```
nit but could we just do this?